src/popup/components/SeedPhraseBadge.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <BtnPlain :class="['seed-phrase-badge', { selected, editable }]">
+    <div>{{ text }}</div>
+    <CloseIcon
+      v-if="editable"
+      class="close-icon"
+    />
+  </BtnPlain>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import BtnPlain from './buttons/BtnPlain.vue';
+import CloseIcon from '../../icons/close.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    BtnPlain,
+    CloseIcon,
+  },
+  props: {
+    selected: Boolean,
+    editable: Boolean,
+    text: { type: String, required: true },
```

Comment:
Is `required` needed? Isn't it required by default.
The first two props should be `selected: Boolean?` and `editable:Boolean?`
Language is typescript then we can use `?` feature for specifying the Maybe null type.

Reply:
Yes, we always need to have the text